frontend/zulu/src/components/CardComponent/CardComponent.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import Card from "../Card/Card";
+import * as PropTypes from "prop-types";
+import axios from "axios";
+
+
+import authHeader from '../../services/auth-header';
```

Comment:
Can you move this to the `story-service` instead and extend it with the needed calls?

Reply:
done